datatypes/src/main/java/org/hyperledger/besu/datatypes/Address.java

```diff
@@ -57,26 +57,31 @@ public class Address extends DelegatingBytes {
   public static final Address ALTBN128_PAIRING = Address.precompiled(0x08);
   /** The constant BLAKE2B_F_COMPRESSION. */
   public static final Address BLAKE2B_F_COMPRESSION = Address.precompiled(0x09);
+  /** The constant KZG_POINT_EVAL aka POINT_EVALUATION_PRECOMPILE_ADDRESS. */
+  public static final Address KZG_POINT_EVAL = Address.precompiled(0xA);
+  /** The constant PARENT_BEACON_BLOCK_ROOT_REGISTRY aka HISTORY_STORAGE_ADDRESS. */
+  public static final Address PARENT_BEACON_BLOCK_ROOT_REGISTRY =
+      Address.precompiled(
+          0xB); // TODO: this is not a precompile anymore. Fix after testnet 8 when we know what the
+  // actual address is.
   /** The constant BLS12_G1ADD. */
-  public static final Address BLS12_G1ADD = Address.precompiled(0xA);
+  public static final Address BLS12_G1ADD = Address.precompiled(0xC);
```

Comment:
are these really all changing?

Reply:
They are not official yet, the BLS ones.